dist/ami/build_ami.sh

```diff
@@ -62,9 +62,8 @@ pkg_install() {
     fi
 }
 
-AMI=ami-ae7bfdb8
```

Comment:
The commit message should explain the extra things we do, like renaming users.

Reply:
Added description at https://github.com/scylladb/scylla/commit/efd0fc9f769e9e6020031bbd9b6bc399e5219800